lang/elixir/elixir.talon

```diff
@@ -0,0 +1,47 @@
+code.language: elixir
+-
+tag(): user.code_functional
+tag(): user.code_concurrent
+
+tag(): user.code_comment_line
+tag(): user.code_data_bool
+tag(): user.code_data_null
+tag(): user.code_functions
+tag(): user.code_keywords
+tag(): user.code_libraries
+tag(): user.code_operators_array
+tag(): user.code_operators_assignment
+tag(): user.code_operators_math
+tag(): user.code_operators_lambda
+
+settings():
+    user.code_private_function_formatter = "snake_case"
+    user.code_public_function_formatter = "snake_case"
+    user.code_private_variable_formatter = "snake_case"
+    user.code_public_variable_formatter = "snake_case"
+
+# Elixir-specific grammars
+state def: "def "
+state defp: "defp "
+state if: "if "
+state else: "else"
+state case: "case "
+state cond: "cond do"
+state try: "try do"
+state rescue: "rescue"
+state after: "after"
+state end: "end"
+
+op pipe: " |> "
+
+# Elixir-specific keywords and symbols
+true: "true"
+false: "false"
+nil: "nil"
+
+[state] raise {user.elixir_exception}: user.insert_between("raise ", "")
+
+[state] rescue {user.elixir_exception}: "rescue {elixir_exception}"
+
+comment line: user.code_comment_line_prefix()
+comment block: user.code_comment_block_prefix()
```

Comment:
`user.code_comment_line`
`user.code_comment_block`

Unclear why the block commit uses "block comment" instead of comment block as the grammar 😭

Reply:
It turns out that elixir does not have multiline comments, which was a surprise to me but I don't think I need to support it now.